content/docs/command-reference/config.md

```diff
@@ -156,27 +156,26 @@ for more details.) This section contains the following options:
   `dvc remote` for more information on "local remotes".) This will overwrite the
   value provided to `dvc config cache.dir` or `dvc cache dir`.
 
-- `cache.ssh` - name of an
-  [SSH remote to use as external cache](/doc/user-guide/managing-external-data#ssh).
```

Comment:
This file was just reordered to match the standard sorting of remote types (implemented in recent PRs) — but these cache.{types} options are directly related to external cache setup, and the docs link to each other.